src/services/routes/org.routes.spec.ts

```diff
@@ -392,24 +392,53 @@ describe('org routes', () => {
       method: 'DELETE',
     };
 
-    test.each([
-      ['ASCII', ORG_NAME],
-      ['Non ASCII', NON_ASCII_ORG_NAME],
-    ])('%s name should be accepted', async (_type, name: string) => {
+    test('Valid name should be accepted', async () => {
+      mockGetOrg.mockResolvedValueOnce({
+        didSucceed: true,
+
+        result: {
+          name: ORG_NAME,
+          memberAccessType: 'INVITE_ONLY',
+        },
+      });
       mockDeleteOrg.mockResolvedValueOnce({
         didSucceed: true,
       });
 
       const response = await serverInstance.inject({
         ...injectionOptions,
-        url: `/orgs/${name}`,
+        url: `/orgs/${ORG_NAME}`,
       });
 
-      expect(mockDeleteOrg).toHaveBeenCalledWith(name, {
+      expect(mockGetOrg).toHaveBeenCalledWith(ORG_NAME, {
+        logger: serverInstance.log,
+        dbConnection: serverInstance.mongoose,
+      });
```

Comment:
I don't think it matters that we used `mockGetOrg` -- what matters is the effect (i.e., 404 response if non-existing), which is already checked in the other test.

The distinction will matter once we eventually replace that `getOrg()` call with a Fastify validator/plugin that checks the existence and auth.